packages/webapp/pages/history.tsx

```diff
@@ -40,7 +39,7 @@ const History = (): ReactElement => {
   const router = useRouter();
   const tabQuery = router.query?.t?.toString() as HistoryType;
   const [page, setPage] = useState(HistoryType.Reading);
-  const { isNewMobileLayout } = useMobileUxExperiment();
+  const isMobileLayout = !isLaptop;
```

Comment:
We don't even need this, we can just use the `isLaptop` alone from the conditionals since we negate them from time to time anyway.